lambda/convertActionNetworkEventCampainToGeoJSON.py

```diff
@@ -45,15 +45,17 @@ def is_us(event: Dict) -> bool:
 def make_location(event: Dict) -> str:
     location = (event.get('location', {}) or {})
     # need to update to handle null values
-    full_location = '{venue}, {address}, {locality}, {region} {postal_code}'.format(
-        venue = safestrip(location.get('venue')), address = safestrip(location.get('address_lines')[0]), locality = safestrip(location.get('locality')),
-        region = safestrip(location.get('region')), postal_code = safestrip(location.get('postal_code'))
-        )
+    full_location = '{address}, {locality}, {region} {postal_code}'.format(
+        address=safestrip(location.get('address_lines')[0]),
+        locality=safestrip(location.get('locality')),
+        region=safestrip(location.get('region')),
+        postal_code=safestrip(location.get('postal_code'))
+    )
     return full_location
 
-def make_state(event: Dict) -> str:
+def parse_location_data(event: Dict, data: str) -> str:
```

Comment:
instead of `data` something like `key` or `dataKey` or `keyName` would be clearer